curriculum/challenges/english/25-front-end-development/workshop-loan-qualification-checker/66c8ba975ee7230e29f6c4b9.md

```diff
@@ -0,0 +1,66 @@
+---
+id: 66c8ba975ee7230e29f6c4b9
+title: Step 18
+challengeType: 1
+dashedName: step-18
+---
+
+# --description--
+
+Now, play around with the `annualIncome` and `creditScore` variables to get any of the condition to pass.
```

Comment:
We should mention that they need to update the values to pass the test here

maybe something like this

```suggestion
Now it is time to test out your loan qualification checker. Update the values for the `annualIncome` and `creditScore` variables to see the different loan messages. You can update the values to whatever positive integers you like.
```